src/executor/JobExecutor.ts

```diff
@@ -55,8 +52,8 @@ export class JobExecutor {
       stopped: this.stopped,
     });
 
-    if (!this.stopped) {
-      await this.executionsRepository.removeExecution(this.scheduleId, jobEntity.name);
+    if (!force && !this.stopped) {
```

Comment:
@YaniKolev Most probably to prevent DB errors. If we call this ->`await this.updateOne({ scheduleId: this.scheduleId }, { $inc: { [`executions.${name}`]: -1 } });` when the execution is already deleted.